WordPress/src/main/java/org/wordpress/android/ui/accounts/login/WPcomLoginHelper.kt

```diff
@@ -26,19 +26,24 @@ class WPcomLoginHelper @Inject constructor(
 
     val wpcomLoginUri = loginClient.loginUri(appSecrets.redirectUri)
     private val customTabsServiceConnection = ServiceConnection(wpcomLoginUri)
+    private var processedAuthData: String? = null
 
-    fun tryLoginWithDataString(data: String?) {
-        if (data == null) {
-            return
+    @Suppress("ReturnCount")
```

Comment:
This seems like a terrible lint. IMHO we should disable it across the whole codebase...